search/views_test.py

```diff
@@ -105,6 +105,13 @@ def test_normal_user_does_not_have_access(self):
         self.client.force_login(user)
         self.assert_status_code(status.HTTP_403_FORBIDDEN)
 
+    def test_enrolled_user_has_access(self):
+        """Normal user without role cannot access"""
+        user = UserFactory.create()
+        ProgramEnrollmentFactory.create(user=user, progra=self.program1)
```

Comment:
`progra` should be `program`... not sure why this doesn't cause an exception with the factory